www/_posts/2016-02-29-docs-improvement.md

```diff
@@ -0,0 +1,39 @@
+--- 
+layout: post 
+author: 
+    name: Raghav Katyal
+    url: https://github.com/rakatyal
+title:  "Significant Documentation Updates"
+categories: announcements 
+tags: news releases docs 
+---
+
+We are happy to announce that we have made significant content updates to our documentation.
+<!--more-->
+
+A while back we conducted a [survey](https://apachecordovabot.typeform.com/to/BCc5co) on the Cordova website asking for feedback on the experience of using Cordova from the developer standpoint. We received more than 900 responses and documentation emerged as one of the major pain points for the average Cordova developer.
+
+Since then we have made efforts to improve the quality of our documentation. To summarize it:
+
+- Validated and corrected most of the current doc content.
+- Re-organized the content for our website and made our table of contents more concise and composed of action-driven items for ease of navigation.
+- Added a 'Reference' section and updated the articles to make them comprehensive and reader-friendly. The [CLI](http://cordova.apache.org/docs/en/latest/cordova-cli/index.html) for example, now lives under the 'Reference' section.
+- Updated the platform guides as single page content following a common structure. For example, the [Android Guide](http://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html) and others now have the following sections:
+	- Requirements and Support
+	- Installing the Requirements
+	- Project Configuration
+	- Debugging
+	- Signing an App
+	- Platform Centered Workflow
+	- Upgrading
+- Moved Core Plugin docs, like [Cordova Plugin Camera](http://cordova.apache.org/docs/en/latest/cordova-plugin-camera/index.html), from Github to be a part of the website. They are now accessible from one place and are easier to read.
+- Fixed a number of Jira bugs related to documentation as a part of improving these articles.
+
+These all are reflected in the new Docs release and we would love to hear your feedback so that we can continue to work on improving it.
+
+It's easy to provide feedback: 
+
+- [File a JIRA](http://cordova.apache.org/contribute/issues.html) using "Component": Docs
```

Comment:
Could you add a link to the [open issues query](https://issues.apache.org/jira/secure/IssueNavigator.jspa?reset=true&mode=hide&jqlQuery=component+%3D+Docs+AND+project+%3D+CB+AND+resolution+%3D+Unresolved+ORDER+BY+priority+DESC) so that people can see what issues are already open?